docs/reference/advanced_configuration.rst

```diff
@@ -33,3 +33,9 @@ Advanced Configuration
                     class: Sonata\ClassificationBundle\Admin\ContextAdmin
                     controller: SonataAdminBundle:CRUD
                     translation: SonataClassificationBundle
+
+        jms_serializer:
+            metadata:
+                directories:
+                    - { name: 'sonata_datagrid', path: "%kernel.project_dir%/vendor/sonata-project/datagrid-bundle/src/Resources/config/serializer", namespace_prefix: 'Sonata\DatagridBundle' }
+                    - { name: 'sonata_classification', path: "%kernel.project_dir%/vendor/sonata-project/classification-bundle/src/Resources/config/serializer", namespace_prefix: 'Sonata\ClassificationBundle' }
```

Comment:
```suggestion
                    -
                        name: 'sonata_datagrid'
                        path: "%kernel.project_dir%/vendor/sonata-project/datagrid-bundle/src/Resources/config/serializer"
                        namespace_prefix: 'Sonata\DatagridBundle'
                    -
                        name: 'sonata_classification'
                        path: "%kernel.project_dir%/vendor/sonata-project/classification-bundle/src/Resources/config/serializer"
                        namespace_prefix: 'Sonata\ClassificationBundle'
```

Reply:
https://symfony.com/doc/3.x/bundles/NelmioApiDocBundle/alternative_names.html

Symfony use this notation, should we too?

Reply:
Anyway this should be move to jms_serializer tutorial. Removed here.